python/libkvikio/pyproject.toml

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+[build-system]
+build-backend = "scikit_build_core.build"
+requires = [
+    "cmake>=3.26.4",
+    "ninja",
+    "scikit-build-core[pyproject]>=0.7.0",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "libkvikio"
+dynamic = ["version"]
+description = "KvikIO - GPUDirect Storage"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+dependencies = [
+    "cmake>=3.26.4",
+    "ninja",
+    "scikit-build-core[pyproject]>=0.7.0",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
```

Comment:
```suggestion
```

Reply:
Pretty sure this is outdated and can now be removed. I believe I mentioned this earlier, but dfg will not delete lists like this that were produced in earlier versions but then removed from dependencies.yaml.